docs/data/material/components/rating/rating.md

```diff
@@ -12,6 +12,19 @@ githubSource: packages/mui-material/src/Rating
 <p class="description">Ratings provide insight regarding others' opinions and experiences, and can allow the user to submit a rating of their own.</p>
 
 {{"component": "@mui/docs/ComponentLinkHeader"}}
+:::warning
```

Comment:
This isn't going to happen, not here at the top of the file 😄

Reply:
Sure thing. Where should i place it then ? At the bottom ? Could you suggest a place for it. Should i use a warning ? 
Let me know i'll add suggested changes and move it around.